lua/various-textobjs/linewise-textobjs.lua

```diff
@@ -141,7 +141,8 @@ end
 ---indentation textobj
 ---@param startBorder "inner"|"outer"
 ---@param endBorder "inner"|"outer"
```

Comment:
the luadoc annotation is missing, which is why the LSP complains

Reply:
Not sure how to make the param optional:

something like this?

`---@param[opt='true'] includeBlankLines "true"|"false"`

Reply:
Looks like it is:

`---@param includeBlankLines? "true"|"false"`